cmd/submit.go

```diff
@@ -240,6 +242,15 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	}
 	defer resp.Body.Close()
 
+	var jsonErrBody apiErrorMessage
+	if err := json.NewDecoder(resp.Body).Decode(&jsonErrBody); err != nil {
+		return fmt.Errorf("failed to parse response - %s", err)
+	}
+
+	if resp.StatusCode != http.StatusOK {
```

Comment:
We probably want to check the entire range of success (2xx). We currently PATCH and expect a 201 response, so this will fail as is.

Reply:
If we expect a 201 response wouldn't it be better to use `http.StatusCreated`?

Reply:
Given that we're decoding an explicit error response, this would be brittle. The API might change to return something other than 201 and this would signal false errors.

I don't want to overstep any boundaries here. Definitely take what I say with a grain of salt until a committer chimes in.

Reply:
👍